.github/workflows/pull_request.yml

```diff
@@ -99,50 +109,18 @@ jobs:
           echo "KUBECONFIG=$HOME/.kube/config" >> $GITHUB_ENV
           echo "OPENEBS_NAMESPACE=openebs" >> $GITHUB_ENV
 
-      - name: Build images locally
-        run: make zfs-driver-image || exit 1;
-
       - name: bootstrap
         run: make bootstrap
 
       - name: Running tests
         run: |
           make ci
           make sanity 
-
+      
       - name: Upload CI Test Coverage Report
         uses: codecov/codecov-action@v4  
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           file: ./tests/bdd_coverage.txt
           name: coverage-bdd_coverage-$(date +%s)
           flags: bddtests
-
-  csi-driver:
-    runs-on: ubuntu-latest
-    needs: ['lint', 'bdd-test']
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Set up QEMU
-        uses: docker/setup-qemu-action@v3
-        with:
-          platforms: all
-
-      - name: Set up Docker Buildx
-        id: buildx
-        uses: docker/setup-buildx-action@v3
-        with:
-          version: v0.13.1
-
-      - name: Build
-        uses: docker/build-push-action@v5
-        with:
-          context: .
-          file: ./buildscripts/zfs-driver/zfs-driver.Dockerfile
-          push: false
-          platforms: linux/amd64, linux/arm64
-          tags: |
-            openebs/zfs-driver:ci
```

Comment:
This workflow tests the multi-arch build. Some CPU instructions may not translate well to other architectures. This workflow tests builds for linux-arm64 as well as linux/am64. Also it tests the multi-stage build. The build before the integration tests is a simpler single-stage build. It's quicker, and there the main intention is to test the code behaviour.

We shouldn't remove this, I think.

Reply:
Okay, but why is the tag needed here?